Libraries/LibWeb/CSS/CSSStyleProperties.cpp

```diff
@@ -148,7 +146,15 @@ String CSSStyleProperties::item(size_t index) const
         return string_from_property_id(property_id).to_string();
     }
 
-    return CSS::string_from_property_id(m_properties[index].property_id).to_string();
+    if (index < custom_properties_count) {
+        auto keys = m_custom_properties.keys();
+        auto custom_property = m_custom_properties.get(keys[index]);
+        if (custom_property.has_value()) {
+            return CSS::string_from_property_id(custom_property.ptr()->property_id).to_string();
+        }
```

Comment:
This `if` shouldn't be needed: If the HashMap can't find a value for its own key, something has gone very wrong. Also in that case, we'd end up falling through to the return below this which would do a lookup with a wrong index.